### PR TITLE
Fixes #29

### LINKED ISSUE #29: Samples tests should pass
Since we have enabled nightly builds for all projects, having all tests pass is important to keep things under control. Could you please take a look at the failing samples tests here https://circleci.com/gh/googleapis/nodejs-monitoring/797 and either fix or disable the failing tests? Thanks!

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: samples/package.json [hunk 1]
```diff
@@ -9,16 +9,16 @@
     "node": ">=4.0.0"
   },
   "scripts": {
-    "test": "repo-tools test run --cmd ava -- -T 3m --verbose system-test/*.test.js"
+    "test": "repo-tools test run --cmd ava -- -T 3m --verbose system-test/uptime.test.js"
```
Comment (alexander-fenster) [inline samples/package.json]: Should we put `*` back?

**Comment by alexander-fenster** [comment]
Let me fix the version problem causing `lint` failure in another branch and then merge it here.

File: samples/package.json (hunk 1 above)
Comment (jmdobry) [reply samples/package.json]: Done

**Comment by alexander-fenster** [comment]
Let's merge #35 first and then I'll resolve conflicts and merge this one as well.

**Pull request merged**